pkgs/development/libraries/lcms2/default.nix

```diff
@@ -1,24 +1,32 @@
-{ lib, stdenv, fetchurl, libtiff, libjpeg, zlib }:
+{ lib, stdenv, fetchFromGitHub, libtiff, libjpeg, nix-update-script, zlib }:
 
 stdenv.mkDerivation rec {
   pname = "lcms2";
-  version = "2.13.1";
-
-  src = fetchurl {
-    url = "mirror://sourceforge/lcms/${pname}-${version}.tar.gz";
-    sha256 = "sha256-1HPnlueyfFrwG9bRVS1CtFtDRX5xgs6ZA/OLt0ggO4g=";
-  };
+  version = "2.14";
 
   outputs = [ "bin" "dev" "out" ];
 
+  src = fetchFromGitHub {
+    owner = "mm2";
+    repo = "Little-CMS";
+    rev = "lcms${version}";
+    sha256 = "sha256-E4D25wRUPNvcaCR9Hx72Hfsb3sc7P5yb8EgfExFxZFg=";
+  };
+
   propagatedBuildInputs = [ libtiff libjpeg zlib ];
```

Comment:
Is this still correct?

Reply:
Looks like yes: https://github.com/mm2/Little-CMS/blob/lcms2.14/configure.ac

Reply:
Sorry, I am not that fluent in autoconf. I was referring to the propagation part.

I don't think the pc file requires those explicitly https://github.com/mm2/Little-CMS/blob/master/lcms2.pc.in